recipes/earcut/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.1)
+project(PackageTest CXX)
+
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
+add_executable(example example.cpp)
+target_link_libraries(example ${CONAN_LIBS})
+set_property(TARGET example PROPERTY CXX_STANDARD 11)
+
+enable_testing()
+add_test(NAME example
+         WORKING_DIRECTORY ${CMAKE_BINARY_DIR}/bin
+         COMMAND example)
```

Comment:
hmmm 🤔 this is different than out usually template... though it looks good we'll see if it's accepted!

Reply:
I actually generated the package template using conan (```conan new --header --test```), and that's where this part came from. In my mind, this makes the test recipe cleaner than the previous approach.

Reply:
Nope, that CTest is commented in CMake file. I remember very well when I added it in Conan and reviewers didn't accept by default. Now I see better and agree, ctest doesn't show the regular output, it's more a testing feature, but test package is more like running an example than creating a unit test.

Reply:
Indeed, it was commented out. Still, anything that's present in a template is likely to strike users as a form of guidance in my opinion. I definitely perceived it as such; and acted on it even though going with the previous approach would've been quicker. Otherwise, your reasoning about using the former approach is perfectly viable. In the end, however, all of this gives way to a bit of confusion: if it isn't the standard way of conducting business, how did it end up in the template?